### DISCUSSION TITLE
Migrating Budibase

### DISCUSSION THREAD

**Discussion by ChadLeJeune** [Budibase/budibase#2893] [General]
Is it possible to migrate a provisioned Budibase to another cloud provider? If I set up Budibase on Digital Ocean, it's up and running and being used, but then down the road I want to host it on GCP or Azure, can I migrate both the applications and the application data?

**Comment by mjashanks** [comment]
Absolutely! We are currently working on a "Settings Export" feature, which will allow you to move all your setup from one server to another. This will be available over the next few weeks.

Then, you can export & import each app, as you need it. This feature is already there.

**Reply by ChadLeJeune** [reply]
Beautiful! And this will export data in the app database(s) as well?

**Reply by mjashanks** [reply]
Yes - it will export the data from Budibase's database. But not your external ones - e.g. SQL - those are up to you. It will save the connection details to external DBs.

**Reply by sibil** [reply]
Thank you Budibase team for the excellent product.

Is the "Settings Export" feature available? I'm trying to move the app to another server and I want to retain the user information so that all users can continue to work with the same credentials.

**Reply by StephanMeijer** [reply]
I cannot find this feature in the User Interface. is that correct?

**Reply by Etkiny** [reply]
@mjashanks hello :) Has this "Settings Export" feature been relased? I would like to move my cloud based account to the self-hosted section. Is it possible?

**Reply by mjashanks** [reply]
Hey @Etkiny - we actually never made a full settings export. Only an "Export Apps", that allows you to export all of your apps at once. You can find this in the Apps List page (at the top)

**Reply by mjashanks** [reply]
Users are a bit of a problem - they can be scripted in via our API though.
